setup.py

```diff
@@ -94,7 +94,7 @@
         "cchecksum>=0.0.3,<1",
         # Pandas peer-dep: Numpy 2.0 causes issues for some users.
         "numpy<2",
-        "packaging>=23.0,<24",
+        "packaging>=24.2,<25",
```

Comment:
We have to fix in some downstream plugins before this will succeed: https://github.com/ApeWorX/ape-hardhat/blob/main/setup.py#L81

Reply:
Actually... This is a bit of a circular dependency. The container depends on both core Ape and plugins. Plugins depend on core ape. This why I think `recommened_plugins` type flows don't make much sense.

The only solution to merge with with a failing solution and then update the plugin and then the next update of core ape will have it all fixed.